core/src/test/scala/stryker4s/report/HtmlReporterTest.scala

```diff
@@ -45,57 +47,75 @@ class HtmlReporterTest extends Stryker4sSuite with MockitoSugar with ArgumentMat
+        """document.querySelector('mutation-test-report-app').report = {"schemaVersion":"1.0","thresholds":{"high":100,"low":0},"files":{}}"""
+      verify(mockFileIO).createAndWrite(testFile, expectedJs)
     }
   }
 
   describe("mutation-test-elements") {
-    it("should find the resource") {
+    it("should write the resource") {
       implicit val config: Config = Config()
       val fileIO = DiskFileIO
+      File.usingTemporaryFile() { tempFile =>
+        val sut = new HtmlReporter(fileIO)
 
-      val sut = new HtmlReporter(fileIO)
-
-      val result = sut.indexHtml()
-      result.mkString.length should be > 50
+        sut.writeMutationTestElementsJsTo(tempFile)
+        val atLeastSize: Long = 200 * 1000 // 200KB
+        tempFile.size should be > atLeastSize
+      }
     }
   }
 
   describe("reportRunFinished") {
+    implicit val config: Config = Config()
+    val stryker4sReportFolderRegex = ".*target(/|\\\\)stryker4s-report-(\\d*)(/|\\\\)[a-z-]*\\.[a-z]*$"
+
     it("should write the report files to the report directory") {
-      implicit val config: Config = Config()
       val mockFileIO = mock[FileIO]
       val sut = new HtmlReporter(mockFileIO)
       val runResults = MutantRunResults(Nil, 50.0, 30.seconds)
 
       sut.reportRunFinished(runResults)
 
-      val indexCaptor = ArgCaptor[File]
-      val reportCaptor = ArgCaptor[File]
-      verify(mockFileIO).createAndWrite(indexCaptor, any[Iterator[Char]])
-      verify(mockFileIO).createAndWrite(reportCaptor, any[String])
-      val paths = List(indexCaptor.value, reportCaptor.value).map(_.pathAsString)
+      val writtenFilesCaptor = ArgCaptor[File]
 
-      // ends with target/stryker4s-report-$TIMESTAMP/filename.extension
-      all(paths) should fullyMatch regex ".*target(/|\\\\)stryker4s-report-(\\d*)(/|\\\\)[a-z]*\\.[a-z]*$"
-      indexCaptor.value.name should be("index.html")
-      reportCaptor.value.name should be("report.js")
+      verify(mockFileIO, times(2)).createAndWrite(writtenFilesCaptor, any[String])
+
+      val paths = writtenFilesCaptor.values.map(_.pathAsString)
+      all(paths) should fullyMatch regex stryker4sReportFolderRegex
+
+      writtenFilesCaptor.values.map(_.name) should contain only ("index.html", "report.js")
+    }
+
+    it("should write the mutation-test-elements.js file to the report directory") {
+      val mockFileIO = mock[FileIO]
+      val sut = new HtmlReporter(mockFileIO)
+      val runResults = MutantRunResults(Nil, 50.0, 30.seconds)
+
+      sut.reportRunFinished(runResults)
+
+      val elementsCaptor = ArgCaptor[File]
+      verify(mockFileIO).createAndWriteFromResource(elementsCaptor,
+                                                    eqTo("mutation-testing-elements/mutation-test-elements.js"))
+
+      elementsCaptor.value.pathAsString should fullyMatch regex stryker4sReportFolderRegex
+      elementsCaptor.value.name equals "mutation-test-elements.js"
     }
 
     it("should debug log a message") {
```

Comment:
should be `"should log a message"`. :)
I agree with changing it to loglevel INFO though.